game/src/main/scala/io/aigar/game/Game.scala

```diff
@@ -0,0 +1,30 @@
+package io.aigar.game
+
+/**
+ * Game holds the logic for an individual game being played
+ * (e.g. the ranked game or a private test game).
+ */
+object Game {
```

Comment:
I have the same concern... Is it to only have one main game maybe sinec the `id` is defined in it?